class.jetpack-gutenberg.php

```diff
@@ -27,6 +27,19 @@ function jetpack_register_block( $slug, $args = array() ) {
 		$slug = 'jetpack/' . $slug;
 	}
 
+	if ( isset( $args['version_requirements'] )
+		&& Jetpack_Gutenberg::is_block_version_gated( $args['version_requirements']['wp'], $args['version_requirements']['plugin'] ) ) {
+		\Jetpack_Gutenberg::set_extension_unavailable(
+			$slug,
+			'incorrect_gutenberg_version',
+			array(
+				'required_feature' => $slug,
+				'required_version' => $args['version_requirements']['plugin'],
```

Comment:
Would we need all the version requirements for the nudge? And would it be beneficial to know the versions detected?